server/backend/database/testcases/testcases.go

```diff
@@ -343,6 +343,43 @@ func RunFindChangesBetweenServerSeqsTest(
 	})
 }
 
+// RunFindChangeInfosBetweenServerSeqsTest runs the FindChangeInfosBetweenServerSeqs test for the given db.
+func RunFindChangeInfosBetweenServerSeqsTest(
+	t *testing.T,
+	db database.Database,
+	projectID types.ID,
+) {
+	t.Run("continues editing without any interference from other users test", func(t *testing.T) {
```

Comment:
I think it would be clearer to focus on testing whether the function works well in both cases (`from > to` and `from <= to`). This approach effectively defines the function's spec as test cases, allowing us to verify that the function performs correctly regardless of any internal optimizations, even if more optimizations are applied later.